tools/benchmark.py

```diff
@@ -51,6 +51,19 @@ def import_data_from_gh_pages():
         write_json(data_file, [])  # writes empty json data
 
 
+def get_binary_sizes(build_dir):
+    path_dict = {
+        "deno": os.path.join(build_dir, "deno"),
```

Comment:
If this runs on windows it will fail here, because it's missing the exe. You can use `util.executable_suffix`.

Reply:
Ooops, thanks for pointing out